components/DownloadPopup.vue

```diff
@@ -82,6 +83,17 @@
             </v-card-actions>
           </v-card>
         </v-dialog>
+
+        <div class="text-text-h3 my-4">
+          Looking for <b>PDF download</b> instead?
+        </div>
+        <v-btn
+          color="#07bc4c"
+          dark
+          @click="$vuetify.goTo('#payButton', { duration: 300, offset: 100 })"
+          ><v-icon class="mr-2">mdi-keyboard-return</v-icon>go to PDF
```

Comment:
a consistent design with the other goto buttons would be cool. 
Maybe the reverse Icon of the download button? 

![save2](https://user-images.githubusercontent.com/23440847/111042753-3613b780-843f-11eb-89c1-dafb03112cc9.png)
![Save1](https://user-images.githubusercontent.com/23440847/111042755-36ac4e00-843f-11eb-802d-c0144fd4fe2f.png)

Reply:
I think that's fine

Reply:
That's not part of this PR.
We should have Standart btns somewhere, but for now everywhere this random hexcode and other params are used